test/channelz_test.go

```diff
@@ -1342,8 +1342,8 @@ func TestCZSubChannelTraceCreationDeletion(t *testing.T) {
 		if len(scm.Trace.Events) == 0 {
 			return false, fmt.Errorf("there should be at least one trace event for subChannel not 0")
 		}
-		if scm.Trace.Events[len(scm.Trace.Events)-1].Desc != "Subchannel Deleted" {
-			return false, fmt.Errorf("the last trace event should be \"Subchannel Deleted\", not %q", scm.Trace.Events[len(scm.Trace.Events)-1].Desc)
+		if got, want := scm.Trace.Events[len(scm.Trace.Events)-1].Desc, "Subchannel Deleted"; got != want {
+			return false, fmt.Errorf("the last trace event should be \"%s\", not %q", want, scm.Trace.Events[len(scm.Trace.Events)-1].Desc)
```

Comment:
`got`?

Reply:
Just a pattern we use in google-cloud-go to compare what we got vs what we wanted to get. Do you prefer a different name?

Reply:
Done!